guides/dev-environment/README.md

```diff
@@ -30,46 +30,6 @@ Verify its installed by running:
 ~$ virtualenv --version
 ```
 
-## Step 3: Cloning cloudify repositories
-
-Its a good idea to create a brand new directory for cloudify source code. In this tutorial we will use ```~/dev/src/cloudify-cosmo```
-
-### SSH Authentication
-
-We recommend [setting up an SSH Key with your Github account](https://help.github.com/articles/generating-ssh-keys/).
-This will allow you to push code by using key-pair authentication, without the need to enter a username and password each time.
-If you do not wish to do so, follow the instructions in the *Basic Authentication* part below.
-
-Clone this repo:
-
-```
-~/dev/src/cloudify-cosmo$ git clone git@github.com:cloudify-cosmo/cloudify-dev.git
-```
-
-Since many repositories need to be cloned, we wrote a small python script to do that for you:
-
-```
-~/dev/src/cloudify-cosmo$ python cloudify-dev/scripts/clone_ssh.py
-```
-
-### Basic Authentication
-
-This method of cloning will require you to authenticate with your Github credentials each time you push code.
-
-Clone this repo:
-
-```
-~/dev/src/cloudify-cosmo$ git clone https://github.com/cloudify-cosmo/cloudify-dev
-```
-
-Since many repositories need to be cloned, we wrote a small python script to do that for you:
-
-```
-~/dev/src/cloudify-cosmo$ python cloudify-dev/scripts/clone_basic.py
-```
-
-## Step 4: Creating a virtualenv
-
 Its also a good idea to have a dedicated directory for python virtual environments. In this tutorial we will use ```~/dev/venvs```
```

Comment:
I would leave this as at is.. (step 4 for creating a virtualenv and step 4.1 for installing python 2.6)